solutions/kubernetes/ceph-deploy-functions.sh

```diff
@@ -103,7 +103,7 @@ function install_ceph() {
         rocky)
             pushd /root/RPMS # subject to change until binaries are fetched from a central repo
                 mv noarch/*.rpm . && mv x86_64/*.rpm . && rmdir noarch/ x86_64/
-                rpm -ivh *.rpm
+                rpm -ivh ./*.rpm
```

Comment:
Please check that the scripts work with this change

Reply:
Remove this file

Reply:
Remove this file